MANIFEST.in

```diff
@@ -4,8 +4,9 @@ include NEWS.rst
 include LICENSE
 include README.rst
 
-recursive-include tests *
 recursive-exclude * __pycache__
 recursive-exclude * *.py[co]
 
 recursive-include docs *.rst conf.py *.jpg *.png *.gif
+
+prune tests
```

Comment:
can't we do `recursive-exclude` instead of prune?

Reply:
sure, I applied it